hyperion/tests/tests_post.py

```diff
@@ -42,25 +42,25 @@ def setUp(self):
         Friend.objects.create(profile1=self.u_1.profile, profile2=self.u_4.profile)
         Friend.objects.create(profile1=self.u_2.profile, profile2=self.u_3.profile)
 
-    def test_private_to_me(self):
-        post = Post.objects.create(
-            author=self.u_1.profile,
-            title="u1_private",
-            content="test1"
-        )
-        post.visible_to_me()
-        self.assertEqual(list(post.visible_to.all()), [self.u_1.profile])
+    # def test_private_to_me(self):
```

Comment:
Why comment out all those tests?

Reply:
If they are not needed anymore, just delete them.

Reply:
Post model has changed a lot and old tests are useless now, those tests may help when I am rewriting the tests. For cleanliness I will delete them.